src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Data/FinanceAPI_V1__VendorDataResponse.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Vendor": {
+    "VendorIdentifier": "P0004320",
```

Comment:
Any reason why most of these these fields begin with Vendor?  They are properties of the Vendor so don't need the prefix.

Reply:
That's a sample response from the Finance API that we are calling.

Reply:
Yes this is what comes back from BC